src/index.html

```diff
@@ -77,13 +77,23 @@
                 name="routerName"
                 placeholder="{{'basicRouterSetup.routerName.placeholder' | translate }}"
                 data-ng-model="wizard.router.name"
+                data-ng-required="(wizard.internet.share && wizard.internet.vpn03.enabled && state.internet.vpn03.generate) || state.ip.register"
                 required pattern="[A-Za-z0-9_\-]{0,16}"
                 >
               <span class="fa form-control-feedback"
                 ng-class="formControlFeedback('routerName')"></span>
-              <p class="text-muted" translate>
+              <p class="text-muted" ng-hide="hasError('routerName')" translate>
```

Comment:
If there is an error the error message is visible instead of the help text. I think showing the error message in addition to the help text too much and not showing any error message or just changing the help text color to red is not enough.  
So I like it very much this way :)